en-US/Players_Guide/04_Adventuring/Combat_Damage_and_Healing.md

```diff
@@ -0,0 +1,190 @@
+
+**Force.**
+Force is pure magical energy focused into a damaging form.
+Most effects that deal force damage are spells, including Magic Missile and spiritual weapon.
+
+**Lightning.**
+A Lightning Bolt spell and a blue dragon’s breath deal lightning damage.
+
+**Necrotic.**
+Necrotic damage, dealt by certain undead and a spell such as Chill Touch, withers matter and even the soul.
+
+**Piercing.**
+Puncturing and impaling attacks, including spears and monsters’ bites, deal piercing damage.
+
+**Poison.**
+Venomous stings and the toxic gas of a green dragon’s breath deal poison damage.
+
+**Psychic.**
+Mental abilities such as a mind flayer’s psionic blast deal psychic damage.
+
+**Radiant.**
+Radiant damage, dealt by a Cleric’s Flame Strike spell or an angel’s smiting weapon, sears the flesh like fire and overloads the spirit with power.
+
+**Slashing.**
+Swords, axes, and monsters’ claws deal slashing damage.
+
+**Thunder.**
+A concussive burst of sound, such as the effect of the Thunderwave spell, deals thunder damage.
+
+##### Damage Resistance and Vulnerability
+
+Some creatures and objects are exceedingly difficult or unusually easy to hurt with certain types of damage.
+
+If a creature or an object has **resistance** to a damage type, damage of that type is halved against it.
+If a creature or an object has **vulnerability** to a damage type, damage of that type is doubled against it.
+
+Resistance and vulnerability apply after other damage modifiers.
+For instance, a creature has bludgeoning resistance and takes 25 bludgeoning damage.
+A magical aura reduces all damage by 5, so the 25 is reduced to 10.
+
+Multiple instances of resistance or vulnerability for the same damage type count as one.
+For example, a creature has bludgeoning and nonmagical damage resistance, so nonmagical fire damage is reduced by half, not three-quarters.
+
+##### Healing
+
+Unless it results in death, damage isn’t permanent.
+Even death is reversible through powerful magic.
+Rest can restore a creature’s hit points, and magical methods such as a [Cure Wounds](#Cure_Wounds_cure_wounds) spell or a potion of healing can remove damage in an instant.
+
+When a creature receives healing of any kind, hit points regained are added to its current hit points.
+A creature’s hit points can’t exceed its hit point maximum, so any hit points regained in excess of this number are lost.
+For example, a Wodewose grants a Ranger 8 hit points of healing.
+If the Ranger has 14 current hit points and has a hit point maximum of 20, the Ranger regains 6 hit points from the Wodewose, not 8.
+
+A creature that has died can’t regain hit points until magic such as the [Revivify](#Revivify_revivify) spell has restored it to life.
+
+##### Dropping to 0 Hit Points
+
+When you drop to 0 hit points, you either die outright or fall unconscious, as explained in the following sections.
+
+###### Instant Death
+
+> **Warning:**
+> This section has not yet been transfered from the Google Doc.
```

Comment:
I did not add this section, since it seems that the Google Doc contains the version from A5E, which is significantly different from the WotC one. So I want to make sure, whether this is intentional. (See the Google Doc for more.)

Once I know what the goal is, I can obviously add whatever version(s) should be added.